jekyll/_cci2/(WIP) api-v2-pipelines-parameters-scopes.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: classic-docs
+title: "API v2 Pipelines, Parameters and Scopes"
+short-title: "API v2 Miscellaneous"
+description: "detailed information about API v2 pipelines, parameters, and scopes"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 was designed to utilize pipelines, parameters, and scopes so you can work with API v2 more efficiently.
+
+## Pipeline Parameters
+
+Pipeline parameters are new with API v2. To use pipeline parameters you must use configuration version 2.1 or higher.
```

Comment:
probably want to link out to how to set that https://circleci.com/docs/2.0/configuration-reference/#version

Reply:
I would also assume they'd need Pipelines enabled....would it be redundant to mention that here?

Reply:
fixed